modules/entity_form/config/schema/entity_browser_entity_form.schema.yml

```diff
@@ -0,0 +1,7 @@
+field.widget.third_party.entity_browser:
```

Comment:
Store entity browser info as third parts setting of IEF widget. There is no UI yet, but it works totally fine if one manually sets stuff in YAML.